core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -715,6 +717,17 @@ public static String getDbVersion() {
         return version;
     }
     
+    public static List<String> getDisabledTabs() {
+        String disabledTabs = "";
+        try {
+            disabledTabs = properties.getProperty(DISABLED_TABS).trim();
+        }
+        catch (NullPointerException e) {}
+        
+        String[] tabs = disabledTabs.split("\\|");
+        return (tabs.length > 0 && disabledTabs.length() > 0) ? Arrays.asList(tabs) : new ArrayList<String>();
```

Comment:
Oops looks like Ben merged before I saw this. @jjgao should we revert and I can make a new PR?